.gitlab-ci.yml

```diff
@@ -15,6 +15,7 @@ stages:
   - TOPOLOGY-CHECK
   - OPENEBS-INSTALL-CHECK
   - OPENEBS-UPGRADE-CHECK
+  - RECOMMENDATION-CHECK
```

Comment:
can we rename this to `CSTORPOOL-RECOMMENDATION-CHECK`

Reply:
Ok @AmitKumarDas